presto-docs/src/main/sphinx/release/release-0.210.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.210
+=============
+
+General Changes
+---------------
+
+* Fix planning failure when aliasing columns of tables containing hidden columns (:issue:`11385`).
+* Fix correctness issue when ``GROUP BY DISTINCT`` terms contain references to the same column
+  using different syntactic forms (:issue:`11120`).
+* Fix failures when querying information_schema tables using capitalized names.
+* Remove min/max intersection heuristics from equality stats calculator.
+* Improve performance when converting between ``ROW`` types.
+* Remove user cpu time tracking.
+* Make cost-based decision when selecting join distribution type.
+
+Hive Changes
+------------
+* Add support for creating Avro Hive tables with schema set using the ``avro_schema_url`` table  property.
+* Support for backward compatible Avro schema evolution.
+* Fix a security bug introduced in 0.209 when using ``hive.security=file``, which would allow any user
```

Comment:
```
* Fix a security bug introduced in 0.209 when using ``hive.security=file``,
  which would allow any user to create, drop, or rename schemas.